homework-3/src/main/java/com/epam/cashier/controller/ProductController.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.cashier.controller;
+
+import com.epam.cashier.controller.dto.ProductDto;
+import com.epam.cashier.controller.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class ProductController {
+    private final ProductService productService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/product")
```

Comment:
Use @RequestMapping("/product") instead of duplicating the same url under each method.

Reply:
fixed